src/assets/small-icons/index.js

```diff
@@ -105,11 +107,12 @@ export { default as twitterIcon } from './ic_twitter.svg';
 export { default as updateIcon } from './ic_update.svg';
 export { default as userCircleIcon } from './ic_user-circle.svg';
 export { default as userIcon } from './ic_user.svg';
+export { default as validationIcon } from './ic_validation.svg';
 export { default as verifiedIcon } from './ic_verified.svg';
 export { default as visibilityOffIcon } from './ic_visibility-off.svg';
 export { default as visibilityOnIcon } from './ic_visibility-on.svg';
 export { default as voiceIcon } from './ic_voice.svg';
 export { default as warningIcon } from './ic_warning.svg';
-export { default as whatappIcon } from './ic_whatapp.svg';
+export { default as whatsappIcon } from './ic_whatsapp.svg';
```

Comment:
Hey, I see this as a potential pitfall! We will be renaming the WhatsApp icon and we should release a major version afterwards!

`customer-app` uses it: https://github.com/homeday-de/customer-app/blob/7ebadee8334b054f4c0399f6b59b6294fc546ac6/src/components/evaluation-report/EvaluationReportSharing.vue#L110

@homeday-de/customer-experience-squad-frontend please be aware of that once updating the library.

Reply:
Hey man! First, thanks for your attention. I'll be sharing this with the team when I merge my changes with deploy. Second, how did you find the customer app's code line? Where did you search the `whatappIcon`? VSCode, github search or something else?

Reply:
GitHub: https://github.com/homeday-de/customer-app/search?q=whatappIcon